README.md

```diff
@@ -61,231 +25,144 @@ For more information on the importance of a professional README for your reposit
-
 <details>
-  <summary>Client</summary>
+  <summary>HTML5</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://youtu.be/x4OKqZ2kIx4">Lesson on HTML5</a></li>
   </ul>
 </details>
 
 <details>
-  <summary>Server</summary>
+  <summary>REACT</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="https://reactjs.org/">Lesson on React.js</a></li>
   </ul>
 </details>
 
 <details>
-<summary>Database</summary>
+<summary>CSS3</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://youtu.be/wIzPMotxMCM">Lesson on CSS3</a></li>
   </ul>
 </details>
 
-<!-- Features -->
-
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Comprehensive Progress Tracking.**
+- **User-Friendly Interface.**
+- **Responsive Design for Mobile Devices.**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
+-  [Live link](https://curriculum-tracker-troos.onrender.com/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
+## 🦻 WalkThrough Video <a name="#walkthrough-demo"></a>
 
-## 💻 Getting Started <a name="getting-started"></a>
+-  [WalkThrough link](https://drive.google.com/file/d/18uOzoGoLmUdF2f-6Fxu_7MCczJEhjZkb/view?usp=sharing)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-> Describe how a new developer could make use of your project.
+## 💻 Getting Started <a name="getting-started"></a>
 
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
+- **To get a proper code editor installed on your computer alternatively you cam use a web browser**
 
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
 
 ### Setup
 
-Clone this repository to your desired folder:
-
-<!--
-Example commands:
-
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
+Clone this repository to your desired folder: https://github.com/Kgomotso196/Curriculum-Tracker.git
 
 ### Install
 
 Install this project with:
-
-<!--
-Example command:
-
-```sh
-  cd my-project
-  gem install
-```
---->
+- Git clone on your cd Desktop using my repository link above
 
 ### Usage
 
-To run the project, execute the following command:
-
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
-
+- To run the project, install necessary dependencies by running `npm install` and `npm start` to open it on the server.
 ### Run tests
 
 To run tests, run the following command:
-
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
+- npx hint .
+- npx stylelint "\*_/_.{css,scss}"
```

Comment:
- [ ] These commands don't run tests, to run tests you use `npm test` the ones you showed are used to run linters instead

Reply:
@descholar-ceo , I will have that fixed. Thanks.